en/advanced_config/land_detector.md

```diff
@@ -30,52 +30,39 @@ Other key parameters that you may need to tune in order to improve landing behav
   
 * [MPC_THR_MIN](../advanced_config/parameter_reference.md#MPC_THR_MIN) - the overall minimum throttle of the system. 
   This should be set to enable a controlled descent.
-
-
-## Fixed Wing Configuration
-
-The complete set of relevant parameters is available under the [LNDFW](../advanced_config/parameter_reference.md#land-detector) prefix. 
-These two parameters are sometimes worth tuning:
-
-* [LNDFW_AIRSPD_MAX](../advanced_config/parameter_reference.md#LNDFW_AIRSPD_MAX) - the maximum airspeed allowed for the system still to be considered landed. 
-  The default of 8 m/s is a reliable tradeoff between airspeed sensing accuracy and triggering fast enough. 
-  Better airspeed sensors should allow lower values of this parameter.
-* [LNDFW_VEL_XY_MAX ](../advanced_config/parameter_reference.md#LNDFW_VEL_XY_MAX) - the maximum horizontal velocity for the system to be still be considered landed. 
-* [LNDFW_VEL_Z_MAX](../advanced_config/parameter_reference.md#LNDFW_VEL_XY_MAX) - the maximum vertical velocity for the system to be still be considered landed.
-  This parameter can be adjusted to ensure land detection triggers earlier or later on throwing the airframe for hand-launches.
-
-
-<span id="states"></span>
-## Land Detector States
-
-### Multicopter Land Detection
+* [MPC_LAND_CRWL](../advanced_config/parameter_reference.md#MPC_LAND_CRWL) - the horizontal and vertical speed limit in the last stage of an automatic landing if the system has a distance sensor. Has to be be set larger than LNDMC_Z_VEL_MAX.
 
 In order to detect landing, the multicopter first has to go through three different states, where each state contains the conditions from the previous states plus tighter constraints.
```

Comment:
@sfuhrer I'm adding a states heading back - firstly because it was linked above, but also because it provides some structure to this otherwise quite monolithic section. OK?

```suggestion

### MC Land Detector States

In order to detect landing, the multicopter first has to go through three different states, where each state contains the conditions from the previous states plus tighter constraints.
```

Reply:
Sure yes :+1: